pages/data.md

```diff
@@ -0,0 +1,38 @@
+---
```

Comment:
Is this page linked anywhere?

Reply:
It's linked in `base.html`, in the footer